lib/menu.rb

```diff
@@ -0,0 +1,25 @@
+require_relative "menu_item"
```

Comment:
Right now, this class has multiple responsibilities:
- it creates a list of menu items by scanning a csv
- it formats those items for display

You might want to consider extracting a Printer class or something similar